log.go

```diff
@@ -75,11 +75,6 @@ func newLogWithSize(storage Storage, logger Logger, maxApplyingEntsSize entryEnc
 	if storage == nil {
```

Comment:
May remove this panic as well. This is obvious, but also the `storage` method calls below would panic anyway.